maven/tests/com.liferay.ide.maven.core.tests/src/com/liferay/ide/maven/core/tests/MavenModuleFragmentProjectTests.java

```diff
@@ -108,6 +114,27 @@ public void testNewModuleFragmentProjectOpProject() throws Exception
             runtime = runtimeWC.save( true, npm );
         }
 
+        File lib =
+            new File( FileLocator.toFileURL( ProjectCore.getDefault().getBundle().getEntry( "lib" ) ).getFile() );
+
+        File tempBladeJar = new File( lib, "com.liferay.blade.cli.jar" );
+
+        boolean needCopy = false;
+
+        if( !tempBladeJar.exists() )
+        {
+            File bladeCliJar = new File(
+                FileLocator.toFileURL(
+                    Platform.getBundle( "com.liferay.ide.maven.core.tests" ).getEntry(
```

Comment:
there is no problem during the build phase, but when we write or debug this test in eclipse side, it will not find the blade jar as it only download during build phase or not deployment env